packages/frontend-2/lib/projects/graphql/queries.ts

```diff
@@ -257,7 +257,6 @@ export const projectAutomationsTabQuery = graphql(`
       }
       ...FormSelectProjects_Project
     }
-    ...ProjectPageAutomationsEmptyState_Query
```

Comment:
Just confirming this, I can't see where the fragment was removed in this branch.

Reply:
It was already removed, but Chuck forgot to remove it here